_includes/nav.html

```diff
@@ -8,15 +8,7 @@
 
     <!-- Logo(s) -->
     <a class="logo-container" href="{{ "/" | relative_url }}">
```

Comment:
I think this needs to be hardcoded to just be `/`. Right now, when I mouse hover logo, it takes me to `/tutorials`. Ideally, we want to take user to homepage (which would be confluent developer).

Reply:
@hartantothio fixed in latest commit: https://github.com/confluentinc/kafka-tutorials/pull/1002/commits/6ad758342feb5c8674d0b2b549d84e4430d6047c